contracts/payment/Escrow.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity ^0.4.23;
+
+import "../math/SafeMath.sol";
+
+
+/**
+ * @title Escrow
+ * @dev Base escrow contract, holds funds destinated to a payee until they
+ * withdraw them.
+ */
+contract Escrow {
+  using SafeMath for uint256;
+
+  mapping(address => uint256) private deposits;
+
+  function depositsOf(address _payee) public view returns (uint256) {
+    return deposits[_payee];
+  }
+
+  /**
+  * @dev Called by the payer to store the sent amount as credit to be pulled.
+  * @param _payee The destination address of the funds.
+  */
+  function deposit(address _payee) payable public {
+    uint256 amount = msg.value;
+    require(amount > 0);
```

Comment:
This line doesn't add any value (ha!). It will unnecessarily propagate the special case to the users of `Escrow`, when we can easily treat `deposit(0)` as a no-op. Let's just remove the line.

Reply:
`deposit(0)` would indeed do nothing: I thought it better to protect the caller from using the API wrong, though the argument could be made that this would make the contract more expensive for all other users. It's more of a library-wide decision, IMO. WDYT?

Reply:
The thing is `deposit(0)` is not a wrong use of the API, it's just a no-op. It's not a matter of gas efficiency, it's about not forcing users of `Escrow` to add code to treat `0` specially. 

We've discussed this before, though I can't remember where, and the decision was to not revert.